<launch>
	<node pkg="ist_object_details" name="ist_object_details" type="ros_object_details" output="screen">
		<rosparam command="load" file="$(find ist_object_details)/config/parameters.yaml"/> 
	</node> 
</launch>

